cmd/snap-repair/runner_test.go

```diff
@@ -1778,3 +1786,77 @@ ls -l ${PATH##*:}/repair
 	c.Assert(err, IsNil)
 
 }
+
+type runner20Suite struct {
+	baseRunnerSuite
+}
+
+var _ = Suite(&runner20Suite{})
+
+var mockModeenv = []byte(`
+model=my-brand/my-model-uc20
+`)
+
+func (s *runner20Suite) SetUpTest(c *C) {
+	s.baseRunnerSuite.SetUpTest(c)
+
+	s.seedAssertsDir = filepath.Join(dirs.SnapSeedDir, "/systems/20201212/assertions")
+
+	// dummy modeenv
+	err := os.MkdirAll(filepath.Dir(dirs.SnapModeenvFile), 0755)
+	c.Assert(err, IsNil)
+	err = ioutil.WriteFile(dirs.SnapModeenvFile, mockModeenv, 0644)
```

Comment:
it would be nice if we also imported boot here and made sure that the mock modeenv we write here is also still a valid modeenv, just to assert that the modeenv is valid, something like:

```go
_, err := boot.ReadModeenv(dirs.SnapModeenvFile)
c.Assert(err, IsNil)
```
right here in SetUpTest would be sufficient I think